src/main/kotlin/org/rust/lang/core/RsPsiPattern.kt

```diff
@@ -84,6 +85,9 @@ object RsPsiPattern {
 
         "non_exhaustive",
 
+        // inner attr
+        "feature",
```

Comment:
I don't think it makes sense to add these comments because you can use, for example, lint attributes (`allow`, `warn`, etc) as inner and outer attributes

Reply:
Oh, this should be part of the realization of another question.